src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindBadGenomicKmersSpark.java

```diff
@@ -54,32 +57,41 @@ protected void runTool( final JavaSparkContext ctx ) {
         SAMSequenceDictionary dict = null;
         if ( hdr != null ) dict = hdr.getSequenceDictionary();
         final PipelineOptions options = getAuthenticatedGCSOptions();
-        final List<SVKmer> killList = findBadGenomicKmers(ctx, getReference(), options, dict);
+        final ReferenceMultiSource referenceMultiSource = getReference();
+        Collection<SVKmer> killList = findBadGenomicKmers(ctx, kSize, minEntropy, referenceMultiSource, options, dict);
+        if ( hasIntervals() ) {
```

Comment:
It'd be good to document somewhere what the intervals mean, since we're just using the standard intervals parameter. Someone new to the tool might think that the intervals are the places over which we'll find non-unique kmers or something similar.